pages/data/index.vue

```diff
@@ -122,7 +122,7 @@
                     <br />
                     <br />
                     <template v-for="dataType in dataTypes">
-                      <dd v-if="resultCounts[dataType] > 0" :key="dataType">
+                      <dd v-if="resultCounts[dataType] > 0 && dataType != 'projects'" :key="dataType">
```

Comment:
Good point, I've gotten lazy about that in other areas of the code too since it usually doesn't matter if its strict or not but for the sake of readability and understanding I should be more explicit